docs/reference/edot-cloud-forwarder/azure.md

```diff
@@ -0,0 +1,284 @@
+
+{{edot-cf}} for Azure uses Bicep to define all the necessary resources and their dependencies.
+
+After you've installed Azure CLI, install Bicep by running the following command:
+
+```bash
+az bicep install
+```
+
+### Managed OTLP endpoint
+
+{{edot-cf}} for Azure requires a {{motlp}} endpoint on {{ecloud}} and an API key.
+
+To find out the URL of the managed OTLP endpoint and the API key for authentication, follow these steps:
+
+::::{dropdown} Steps to retrieve the OTLP endpoint and API key
+
+:::{include} ../_snippets/serverless-endpoint-api.md
+:::
+
+In the Bicep templates, the OTLP endpoint is set as `elasticsearchOtlpEndpoint`, and the API key is set as `elasticsearchApiKey`. 
+
+:::{important}
+Trim the API key from `Authorization=ApiKey MYKEYVALUE...` to just `MYKEYVALUE...` before using it as the argument to the `elasticsearchApiKey` parameter.
+:::
+::::
+
+## Deployment considerations
+
+Before deploying {{edot-cf}} for Azure, keep these points in mind:
+
+Before deploying {{edot-cf}} for Azure, take the following into consideration:
+
+- The logs event hub supports Azure resource logs, and the metrics event hub supports DCR metrics.
+- Support for additional encoding extensions for logs and metrics will be added in future releases.
+
+## Deploy EDOT CF for Azure
+
+Follow these steps to deploy {{edot-cf}} for Azure in a resource group to send logs and metrics to the {{motlp}} endpoint.
+
+::::::{stepper}
+
+:::::{step} Set the environment variables
+Define the following environment variables:
+
+```sh
+export AZURE_LOCATION="<your_region>"
+export RESOURCE_GROUP="<group_name>"
+export ELASTICSEARCH_OTLP_ENDPOINT="<your_otlp_endpoint>"
+export ELASTICSEARCH_API_KEY="<your_api_key>"
+```
+:::::
+
+:::::{step} Create the resource group
+Create the resource group that hosts all the resources for {{edot-cf}} for Azure.
+
+```sh
+$ az group create --name $RESOURCE_GROUP --location $AZURE_LOCATION
+{
+  "id": "/subscriptions/12cabcb4-86e8-404f-a3d2-1dc9982f45ca/resourceGroups/<group_name>",
+  "location": "<your_region>",
+  "managedBy": null,
+  "name": "<group_name>",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+:::::
+
+:::::{step} Download the template
+Download the Bicep template to deploy {{edot-cf}} for Azure at the following URL: 
+
+* https://edotcfazure5gdoxpg7d2rim.blob.core.windows.net/releases/v0.6.0/ecf.bicep
+:::::
+
+:::::{step} Deploy the Bicep template
+Deploy the Bicep template using the following command:
+
+```sh
+az deployment group create \
+    --resource-group ${RESOURCE_GROUP} \
+    --template-file ecf.bicep \
+    --parameters \
+        elasticsearchOtlpEndpoint=$ELASTICSEARCH_OTLP_ENDPOINT \
+        elasticsearchApiKey=$ELASTICSEARCH_API_KEY \
+        logsDecoder=ds \ <1>
+        metricsDecoder=dcr \ <2>
+        eventHubPartitionCount=8 \
+        eventHubMessageRetentionInDays=1 \
+        version=0.6.0 <3>
```

Comment:
@zmoog So using https://ela.st/edot-cf-azure-template we can omit the version field from this command?